pkg/services/ngalert/prom/convert.go

```diff
@@ -34,7 +35,7 @@ var (
 		FromTimeRange:    &defaultTimeRange,
 		EvaluationOffset: &defaultEvaluationOffset,
 		ExecErrState:     models.ErrorErrState,
-		NoDataState:      models.NoData,
+		NoDataState:      models.OK,
```

Comment:
Changed the default value for NoDataState to OK because this is the Prometheus behavior.